flow/probes/ebpf.go

```diff
@@ -207,7 +208,44 @@ func (p *EBPFProbe) flowFromEBPF(ebpfFlow *EBPFFlow, kernFlow *C.struct_flow, up
 
 	return f
 }
-
+func updateFromEBPF(ebpfFlow *EBPFFlow, kernFlow *C.struct_flow){//Maintaining a huge flow table that is accessed via syscalls is expensive.
+                                                                 //Therefore, we remove every flow after extracting its data and we keep
+                                                                 //a local copy that we update continuously to preserve correctness.
+	if ebpfFlow.Kflow.link_layer._hash_src == kernFlow.link_layer._hash_src{
+		kernFlow.metrics.ab_packets+=ebpfFlow.Kflow.metrics.ab_packets
```

Comment:
what about just ebpfFlow.Kflow.metrics.ab_packets+=... 

removing this line 
https://github.com/skydive-project/skydive/pull/1558/files#diff-733c12abf4ac0cf47dc4a9dddd3b9c40R320

Reply:
Took care of this